### ISSUE TITLE
Publicize other ways of teaching the material beyond the standard core lessons.

### ISSUE THREAD

**Issue by jt14den** [LibraryCarpentry/governance#25]
Describe and publicize other ways of teaching the material beyond the standard workshop/core lessons: https://librarycarpentry.org/advisory/. Write a blog entry. 

Talk about the goals for a workshop and let that determine how you mix and match lessons with learning objectives in mind. (backward design).
Check out/summarize https://librarycarpentry.org/lc-overview/ 
Look at suggested recipes, e.g. http://swcarpentry.github.io/r-novice-gapminder/guide/ as a poss. example. 
Look at how LC has been taught in the past (one day composition of lessons, two day, etc.) as poss. guide (how LC is actually taught).

**Comment by libcce** [comment]
Thanks for starting this off @jt14den. Here is some data up until October of what lessons are being taught:

OpenRefine, 86
Shell, 77
Introduction to Data, 73
Git, 37
Python, 34
Tidy Data (Spreadsheets), 27
regex, 17
Jargon Busting, 12
SQL, 10
Git/GitHub, 4
R, 4
Instructor Training, 2
Introduction to LC, 1
Collecting Data Online,1
Consortium, 1
Data Structures, 1 
DST4L, 1
FAIR, 1
Lesson Sprint, 1
Mapping, 1
Visualization Tools, 1
ResBaz, 1
Workflow Enhancement Tools, 1

**Comment by libcce** [comment]
And how many days they are taught:

two-day	75
one-day	50
multi-day	7
half-day	5
other	3
3 half-days	1
three-day	1

**Comment by libcce** [comment]
The trend I see right away is that Tidy Data is growing in popularity and we can benefit from moving it to stable.

**Comment by ragamouf** [comment]
I would support that - although it needs a bath and a scrub up. I think the 2 hour time allowance is too short, and now that I'm going through instructor certification, would probably make different choices about how to teach that.

**Comment by jt14den** [comment]
Thanks @libcce and @ragamouf . Another source of data is the spreadsheet we kept of how we taught LC through 2018, that captures some of the sequences. https://docs.google.com/spreadsheets/d/17EMzgODvvX4qnfpNvHkdIi3VzZp3r_sdYHld1YrpB-A/edit#gid=0. I'd like to update the sequences of episodes taught (or when gone off book). @libcce do you have that data? I suspect we'd need to comb through the workshop websites? 

I now typically swap in tidy data in place of regex @ragamouf. I've had hosts and librarians not want regex, so would be good to see it made core and an optional lesson path.

**Comment by annajiat** [comment]
Due to COVID as well as the workshops being online, 
we might have had additional formats and redistribution in the number of days taught.

**Comment by Fillstar** [comment]
as a newbie - I would be happy to be involved in adding some value to this one
 :)

**Comment by libcce** [comment]
@Fillstar @annajiat I had my hands on both data from AMY and a file I kept to track the different types of workshops back in the day. Since, everything has moved to AMY. Not everything was in AMY a while back, The Carpentries workshop/certification database, and I have a feeling that it still has gaps. The trick here is that a number of these workshops can fall into the Carpentries-based/self run workshops that don't wind up in AMY. I'd hope that The Carpentries keeps some minimal data about those types of workshops (fingers crossed). We can ask on the next LC Advisory Group call as I think @kariljordan will be there. But I think we may need to run a survey to see if we can capture the additional workshops that have been run since we looked at this data (that are possibly not in AMY).
